src/mame/netronics/exp85.cpp

```diff
@@ -134,17 +124,17 @@ void exp85_state::exp85_mem(address_map &map)
 	// Microsoft Basic ROM
 	map(0xc000, 0xdfff).rom();
 	// monitor ROM in the 8355 chip.
-	map(0xf000, 0xf7ff).r(I8355_TAG, FUNC(i8355_device::memory_r));
+	map(0xf000, 0xf7ff).r("u105", FUNC(i8355_device::memory_r));
```

Comment:
You can use the m_ objects here instead of the names.  That avoids additional run-time string lookups, and also eliminates the need to know the name in more than one place.  So just m_rom instead of "u105", and m_i8155 instead of "u106".

Reply:
I agree, the code is much better this way. Anyways, I will keep an 8355 object and a rom object finder, even if the rom is in principle already contained in the 8355 device and I have a reference to it. This would be much simpler if the 8355 class implemented the device_memory_interface interface, since in this case I would be able to get a memory base pointer without the need of a memory region object finder. Do you think it would be reasonable in the future to inherit from this interface in the 8355 device class?